automation/active_user_input/english_working-groups-config.yml

```diff
@@ -16,36 +16,41 @@ end-matter:
 # WG-Common
 
 wg-common:
-  include-wg: False
+  include-wg: True
   wg-fullname: Common Metrics WG
   repo-link: https://github.com/chaoss/wg-common
-  repo-branch: master
+  repo-branch: main
   focus-areas-location: wg-common/focus-areas
   focus-areas:
     contributions:
+    - clones.md
+    - language-distribution.md
     - technical-fork.md
     - types-of-contributions.md
     time:
```

Comment:
I don't know if it's intentional to put the focus area `time` before `people` and `place` considering that it doesn't follow the alphabetical order. I see the same kind of issue at a couple of other places. Could you please confirm?

Reply:
This makes sense. Thanks for clearing the doubt!